test/common/crypto/verify_signature_fuzz_test.cc

```diff
@@ -0,0 +1,43 @@
+#include "common/common/hex.h"
+#include "common/crypto/utility.h"
+#include "common/protobuf/utility.h"
+
+#include "test/common/crypto/verify_signature_fuzz.pb.validate.h"
+#include "test/fuzz/fuzz_runner.h"
+#include "test/test_common/utility.h"
+
+// #include "common/common/logger.h"
+
+namespace Envoy {
+namespace Common {
+namespace Crypto {
+namespace {
+
+DEFINE_PROTO_FUZZER(const test::common::crypto::VerifySignatureFuzzTestCase& input) {
+  // Exit early if the protobuf input does not satisfy the constraints.
```

Comment:
There are no constrains in the proto to validate. Should there be? I like the idea of testing against malformed / bad length / bad functions.

Reply:
> There are no constrains in the proto to validate. Should there be? I like the idea of testing against malformed / bad length / bad functions.

Thanks for pointing this out! I have removed these lines.